src/003-building-a-crate-for-avr.md

```diff
@@ -48,7 +48,7 @@ The ELF file will be available at `target/<TARGET NAME>/release/<CRATE NAME>.elf
 
 Notes:
 
-  * **`-Z build-std=core` is required whenever AVR is being targeted**. See [3.1. A note about the required Rust `-Z build-std=<CRATE,>` flag](./003.1-note-about-rust-build-std-flag.md) for more details.
+  * **`-Z build-std=core` is required whenever AVR is being targeted**. See [3.2. A note about the required Rust `-Z build-std=<CRATE,>` flag](./003.2-note-about-rust-build-std-flag.md) for more details.
   * `--release`  is not strictly required - debug mode should be as correct as release mode - however, **debug mode generates SLOW CODE, especially on AVR**. Release mode is much better.
 
 **Example**: An in-context example of compiling a crate is given for the LED blinking example in [3.2. Example - Building the `blink` program for AVR](./003.2-example-building-blink.md).
```

Comment:
This should also be 3.3

Reply:
Good catch fixed in 9b1c6e9e0adf24f8b532f7743fcafc02ba06a532